### PR TITLE
chore(layout): fix codesandbox example and add code example to Web-components layout README

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

[Layout]: Deprecation of Layout #7328

### Description

Fix layout codesandbox example and link to Web Components Layout README as well.

Add code snippet example to Web Components README

### Changelog

**New**

- Add code example to Web Components Layout Readme

**Changed**

- fix and ensure layout codesandbox works

### CODE REVIEW

File: packages/web-components/src/components/layout/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
+
+##### CSS
 
 ```css
+// yourapplication.scss
+@import '@carbon/grid/scss/grid.scss';
 @import '@carbon/ibmdotcom-styles/scss/components/layout/layout';
 ```
 
+##### JS
```
Comment (jeffchew) [inline packages/web-components/src/components/layout/__stories__/README.stories.mdx:22]: Should this be `HTML` instead?
```suggestion
##### HTML
```

Comment (annawen1) [reply packages/web-components/src/components/layout/__stories__/README.stories.mdx:22]: yep you're right

**Review by jeffchew — APPROVED** [review]
LGTM!

**Comment by jeffchew** [comment]
@annawen1 will wait for the codesandbox e2e tests to run before adding the merge label to this.

**Pull request merged**